docs/reference/metricbeat/exported-fields-system.md

```diff
@@ -871,6 +871,22 @@ type: object
 type: object
 
 
+## ntp [_ntp]
```

Comment:
Same here. This _should_ have an `applies_to` indicating that this was added in a post-9.0.0 release, but we'll have to find a long-term solution for this outside the scope of this PR.